pkg/sql/catalog/bootstrap/bootstrap_test.go

```diff
@@ -164,7 +164,7 @@ func TestSystemDatabaseSchemaBootstrapVersionBumped(t *testing.T) {
 
 	// If you need to update this value (i.e. failed this test), check whether
 	// you need to bump systemschema.SystemDatabaseSchemaBootstrapVersion too.
-	const prevSystemHash = "cd85ebe773d840ecaf39fd4ccbba6345f54d7414a95b5bf5be2fa4102c26c650"
+	const prevSystemHash = "7d089e4c69d8d6b164b3d1a04e0f3ad5f487f9138c9585bd00b37f397691a090"
```

Comment:
Is it ok to bump this without updating `ystemschema.SystemDatabaseSchemaBootstrapVersion` in this case? The system schema should have no UDFs, AFAIK, so I thought just bumping this would be fine.